### ISSUE TITLE
deployment does not respect vhost template

### ISSUE THREAD

**Issue by Schnaaf** [zend-patterns/ZendServerSDK#90]
Hi,

Not sure if this is the right place to post this issue.

If I define the DocumentRoot in my vhost template as follows:
`    DocumentRoot "${docroot}/../myotherdocroot/"`
Then I can create this vhost correctly using the API

Next if I deploy a zpk (with the docroot set in the deployment.xml) the complete value between double quotes gets replaced with the docroot setting from my deployment.xml ignoring how I defined this in my vhost template.

So in short, after deploy I end up with:
`    DocumentRoot "/usr/local/zend/var/apps/<scheme>/<vhost>/<port>/<app>/<docroot_from_deployment.xml>/"`
But in my understanding of a template I should have ended up with:
`    DocumentRoot "/usr/local/zend/var/apps/<scheme>/<vhost>/<port>/<app>/<docroot_from_deployment.xml>/../myotherdocroot/"`

same thing happens with
`<Directory "${docroot}/../myotherdocroot/">`

**Comment by Schnaaf** [comment]
You may be wondering why I want to do this. I'm using this documnted workaround [https://support.zend.com/hc/en-us/articles/203959473](url) to deploy two vhosts within one template where each vhost should have its own document root within the deployed app. If I were to deploy one single vhost then it would be fixable by setting the correct docroot in deployment.xml. Since I want both vhosts to have a different docroot this fix is not helpfull

**Comment by Schnaaf** [comment]
As I've learned from Zend Support this issue is due to Deployment Daemon behavior and has been filed as a bug.
